templates/publisher/listing.html

```diff
@@ -318,6 +318,36 @@
             </div>
           </div>
         </div>
+
+        <div class="p-strip is-shallow">
+          <div class="row">
+            <hr class="u-no-margin--bottom" />
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-2">
+            <label>License</label>
+          </div>
+          <div class="col-8">
+            <input type="hidden" name="license" value="{{ license }}" />
+            <input type="radio" name="license-type" value="simple" id="license-simple-option" {% if license_type == 'simple' %}checked="checked"{% endif %} />
+            <label for="license-simple-option">Simple</label>
```

Comment:
I'm not sure if 'Simple' is a good name for this. Do we need to name this checkbox at all?

<img width="1045" alt="screen shot 2018-10-02 at 13 05 17" src="https://user-images.githubusercontent.com/83575/46345233-178d1d80-c644-11e8-8438-94d4db3c233b.png">